src/lib/swapper/swappers/OsmosisSwapper/utils/helpers.ts

```diff
@@ -233,19 +253,7 @@ export const performIbcTransfer = async (
   }
 }
 
-// TODO: move to chain adapters
-export const buildTradeTx = async ({
-  osmoAddress,
-  adapter,
-  accountNumber,
-  buyAssetDenom,
-  sellAssetDenom,
-  sellAmount,
-  gas,
-  feeAmount,
-  feeDenom,
-  wallet,
-}: {
+type BuildTradeTxInput = {
```

Comment:
personally im a fan on inlining type definitions unless they're shared because it makes them ever so slightly easier to change (one less indirection)

Reply:
ah i see this is shared below in `buildTradeTx`, gotit.